HarfangLab/harfanglab/models.py

```diff
@@ -29,3 +29,15 @@ class JobTriggerResult(BaseModel):
     action: str
     creationtime: str
     parameters: Optional[Dict[str, Any]]
+
+
+class HostnameEntry(BaseModel):
+    hostname: str
+    ipaddress: str
+    last_seen: str
+    os: str
```

Comment:
According to the Harfanglab documentation,
the name of the date when the agent was last seen is `lastseen`.
With the property `os`, do you mean `ostype`?

First suggestion: Rename the properties (don't forget to rename the usage also)
```suggestion
    lastseen: str
    ostype: str
```

Second suggestion: Use Field's alias
```suggestion
    last_seen: str = Field(..., alias='lastseen')
    os: str = Field(..., alias='ostype'
```